pkg/crypto/crypto.go

```diff
@@ -0,0 +1,77 @@
+/*
+	Package crypto provides functions to encode and decode data
+	Using aes crypto
+	Random flag defines random nonce sequence or same
+*/
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"fmt"
+	"io"
+)
+
+type cryptographer struct {
+	Key        []byte
```

Comment:
Make, please, fields of structure as non-exported to forbid to change them outside of the package functionality.